---
title: "How can I use a LRG record?"
faq_tags:
  - most_frequent
  - records
---

Once a LRG record has been created, you can e.g.:  

* View your LRG and all known variants in genome browsers supporting LRGs:

  * **Ensembl**  
    Once an LRG has been released, it is integrated into the next Ensembl release, normally within two months. Public LRGs can be viewed in a set of dedicated pages (for example, [LRG_1](http://www.ensembl.org/Homo_sapiens/LRG/Summary?lrg=LRG_1)) whereas pending LRGs can be viewed by following the Ensembl link in the updatable section of each LRG (for example, [LRG_750](http://www.ensembl.org/Homo_sapiens/Location/View?contigviewbottom=lrg_transcript%3Dtranscript_label;db=core;lrg=LRG_750;r=19:4088321-4129129)).  
  
  * **NCBI** 
    The records are available from NCBI's Nucleotide and Gene databases and can be found in the RefSeqGene browser page and FTP site ([ftp://ftp.ncbi.nlm.nih.gov/refseq/H_sapiens/RefSeqGene/](ftp://ftp.ncbi.nlm.nih.gov/refseq/H_sapiens/RefSeqGene/)). More information on ways to find LRGs at NCBI can be found here [http://www.ncbi.nlm.nih.gov/refseq/rsg/lrg/](http://www.ncbi.nlm.nih.gov/refseq/rsg/lrg/).  

  * **UCSC** 
  The records are available in UCSC Genome Browser at [https://genome.ucsc.edu/](https://genome.ucsc.edu/) by entering the LRG identifier or gene symbol in the search box or by turning on the LRG track under Genes and Gene Predictions.  

* Use **[Mutalyzer](https://mutalyzer.nl/)** to check that variants described using your LRG follow HGVS conventions - Mutalyzer's *"Name Checker"*, *"Syntax Checker"*, and *"Name Generator"* ensure that variants described using LRG sequences follow HGVS guidelines.  

* **Submit your variant data** into a public archive: â€¨ Variants can be submitted to dbSNP ([http://www.ncbi.nlm.nih.gov/SNP/](http://www.ncbi.nlm.nih.gov/SNP/) and ClinVar ([http://www.ncbi.nlm.nih.gov/clinvar/](http://www.ncbi.nlm.nih.gov/clinvar/)) via the [LRG submission process](/submit-variants).  

* Use **[Ensembl's Variant Effect Predictor](http://www.ensembl.org/info/docs/tools/vep/index.html)**, **[Alamut's Interactive Biosoftware](http://www.interactive-biosoftware.com/)**, and **[Variobox](http://bioinformatics.ua.pt/software/variobox/)** to interpret variation data described using your LRG.