<div class="section-double">

  <div class="section-title clearfix">
    <div class="icon-target close-icon-0 section-sub-title-icon left"></div>
    <div class="section-sub-title left">What is unique about LRG reference sequence records?</div>
  </div>

  <div class="section-content padding-top-0 padding-left-0">
    <div class="clearfix">
      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 items_list">
        
        <div class="item_entry_no_border">
          <div class="item_title_no_border">Created specifically for clinical reporting by manual curation</div>
          <div class="item_content_no_border">
            <ul>
              <li>Include a minimum set of transcripts for reporting at a locus (ideally one)</li>
              <li>Manually curated by expert scientists</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border">Stable</div>
          <div class="item_content_no_border">
            <ul>
              <li>Include transcript sequences that are stable and independent of changes to transcript models (<a class="icon-external-link" href="{{ site.urls.refseq }}" target="_blank">RefSeq</a> and <a class="icon-external-link" href="{{ site.urls.gencode }}" target="_blank">GENCODE</a>)</li>
              <li>Establish a coordinate system that is independent from upgrades to the reference genome assembly, and provides mappings to present and past assemblies</li>
              <li>Use a unique and stable identifier that is not versioned</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border">Flexible and collaborative</div>
          <div class="item_content_no_border">
            <ul>
              <li>Allow inclusion of legacy or community-requested reference sequences</li>
              <li>Individual records created at the request of the clinical community and in collaboration with gene-specific experts</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border">Connect the past, present and future of clinical variant reporting</div>
          <div class="item_content_no_border">
            <ul>
              <li>Define the relationship between legacy and community-requested sequences and the reference assembly</li>
              <li>Facilitate unambiguous multi-directional data exchange between RefSeq, GENCODE and the reference genome assemblies (GRCh37, GRCh38)</li>
              <li>Compatible with genome reference assembly-based NGS variant reporting systems</li>
            </ul>
          </div>
        </div>

        <div class="margin-bottom-0">
          <div class="item_title_no_border">Well-supported</div>
          <div class="item_content_no_border">
            <ul class="margin-bottom-0">
              <li>Integrated into the <a class="icon-external-link" href="{{ site.urls.ensembl }}" target="_blank">Ensembl</a>, <a class="icon-external-link" href="{{ site.urls.ncbi }}" target="_blank">NCBI</a> and <a class="icon-external-link" href="{{ site.urls.ucsc }}" target="_blank">UCSC</a> genome browsers to allow visualization in genomic context, with all other existing annotations</li>
              <li>Compatible with the Human Genome Variation Society (HGVS) nomenclature and supported by nomenclature checker systems (e.g. <a class="icon-external-link" href="{{ site.urls.mutalyzer }}" target="_blank">Mutalyzer</a>, <a class="icon-external-link" href="{{ site.urls.vvalidator }}" target="_blank">VariantValidator</a>)</li>
              <li>Generated and maintained by the <a class="icon-external-link" href="{{ site.urls.ncbi }}" target="_blank">NCBI</a> and <a class="icon-external-link" href="{{ site.urls.embl_ebi }}" target="_blank">EMBL-EBI</a></li>
            </ul>
          </div>
        </div>

      </div>
    </div>
  </div>

</div>
