<div class="section-double">

  <div class="section-title section-title-underline clearfix">
    <div class="icon-next-page close-icon-0 left" style="margin-left:6px"></div>
    <div class="icon-next-page close-icon-0 left" style="margin: 0px -5px"></div>
    <div class="icon-next-page close-icon-0 left" style="margin-right:6px"></div>
    <div class="section-sub-title left">LRGs and reporting variants</div>
  </div>

  <div class="section-content">
    <div class="margin-bottom-5">LRGs are suitable for reporting variants because they are:</div>
    <div class="clearfix">
      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
        
        <div class="item_entry_no_border">
          <div class="item_title_no_border close-icon-5 icon-collapse-closed" title="Click to expand" id="item1_button" onclick="javascript:show_hide('item1')">
            Created specifically for clinical reporting by manual curation
          </div>
          <div class="item_content_no_border" id="item1">
            <ul>
              <li>Include a minimum set of transcripts for reporting at a locus (ideally one)</li>
              <li>Manually curated by expert scientists</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border close-icon-5 icon-collapse-closed" title="Click to expand" id="item2_button" onclick="javascript:show_hide('item2')">
            Stable
          </div>
          <div class="item_content_no_border" id="item2">
            <ul>
              <li>Include transcript sequences that are stable and independent of changes to transcript models (RefSeq and GENCODE)</li>
              <li>Establish a coordinate system that is independent from upgrades to the reference genome assembly, and provides mappings to present and past assemblies</li>
              <li>Use a unique and stable identifier that is not versioned</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border close-icon-5 icon-collapse-closed" title="Click to expand" id="item3_button" onclick="javascript:show_hide('item3')">
            Flexible and collaborative
          </div>
          <div class="item_content_no_border" id="item3">
            <ul>
              <li>Allow inclusion of legacy or community-requested reference sequences</li>
              <li>Individual records created at the request of the clinical community and in collaboration with gene-specific experts</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border close-icon-5 icon-collapse-closed" title="Click to expand" id="item4_button" onclick="javascript:show_hide('item4')">
            Connect the past, present and future of clinical variant reporting
          </div>
          <div class="item_content_no_border" id="item4">
            <ul>
              <li>Define the relationship between legacy and community-requested sequences and the reference assembly</li>
              <li>Facilitate unambiguous multi-directional data exchange between RefSeq, GENCODE and the reference genome assemblies (GRCh37, GRCh38)</li>
              <li>Compatible with genome reference assembly-based NGS variant reporting systems</li>
            </ul>
          </div>
        </div>

        <div class="item_entry_no_border">
          <div class="item_title_no_border close-icon-5 icon-collapse-closed" title="Click to expand" id="item5_button" onclick="javascript:show_hide('item5')">
            Well-supported
          </div>
          <div class="item_content_no_border" id="item5">
            <ul>
              <li>Integrated into the Ensembl, NCBI and UCSC genome browsers to allow visualization in genomic context, with all other existing annotations</li>
              <li>Compatible with HGVS nomenclature and supported by nomenclature checker systems (Mutalyzer)</li>
              <li>Generated and maintained by the NCBI and EMBL-EBI</li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>

  <!-- <div class="section-content">
    <div>LRGs are suitable for reporting variants because they are:</div>
    <div class="clearfix">
      <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
        <h5 class="lrg_dark bold_font" style="margin:10px 0px 5px">Created specifically for clinical reporting by manual curation</h5>
        <ul style="margin-top:0px">
          <li>Include a minimum set of transcripts for reporting at a locus (ideally one)</li>
          <li>Manually curated by expert scientists</li>
        </ul>
        <h5 class="lrg_dark bold_font" style="margin:10px 0px 5px">Stable</h5>
        <ul style="margin-top:0px">
          <li>Include transcript sequences that are stable and independent of changes to transcript models (RefSeq and GENCODE)</li>
          <li>Establish a coordinate system that is independent from upgrades to the reference genome assembly, and provides mappings to present and past assemblies</li>
          <li>Use a unique and stable identifier that is not versioned</li>
        </ul>
        <h5 class="lrg_dark bold_font" style="margin:10px 0px 5px">Flexible and collaborative</h5>
        <ul style="margin-top:0px">
          <li>Allow inclusion of legacy or community-requested reference sequences</li>
          <li>Individual records created at the request of the clinical community and in collaboration with gene-specific experts</li>
        </ul>
      </div>
      <div class="col-xs-6 col-sm-6 col-md-6 col-lg-6">
        <h5 class="lrg_dark bold_font" style="margin:10px 0px 5px">Connect the past, present and future of clinical variant reporting</h5>
        <ul style="margin-top:0px">
          <li>Define the relationship between legacy and community-requested sequences and the reference assembly</li>
          <li>Facilitate unambiguous multi-directional data exchange between RefSeq, GENCODE and the reference genome assemblies (GRCh37, GRCh38)</li>
          <li>Compatible with genome reference assembly-based NGS variant reporting systems</li>
        </ul>
        <h5 class="lrg_dark bold_font" style="margin:10px 0px 5px">Well-supported</h5>
        <ul style="margin-top:0px">
          <li>Integrated into the Ensembl, NCBI and UCSC genome browsers to allow visualization in genomic context, with all other existing annotations</li>
          <li>Compatible with HGVS nomenclature and supported by nomenclature checker systems (Mutalyzer)</li>
          <li>Generated and maintained by the NCBI and EMBL-EBI</li>
        </ul>
      </div>
    </div>
  </div>-->
</div>